Hypeclass/API/SearchManager.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  SearchManager.swift
+//  Hypeclass
+//
+//  Created by Hong jeongmin on 2022/07/26.
+//
+
+import Foundation
+import FirebaseFirestore
+import FirebaseFirestoreSwift
+
+class SearchManager {
+    static let shared = SearchManager()
+    
+    func requestSearchDancer(dancerSearch: String) async throws -> [Dancer]? {
```

Comment:
SearchManager 안에있는 메서드라서 `Search` 는 빼도 유추가능할 것 같습니다 ㅎ
파라미터 이름 제안드려요! ㅎㅎ
```suggestion
    func requestDancersBy(queryString query: String) async throws -> [Dancer]? {
```

Reply:
SearchManager 내부라서 query 가 더 적당해 보이네요! 
반영 하였습니다